elm/config/dask_settings.py

```diff
@@ -11,7 +11,7 @@
 from multiprocessing.pool import ThreadPool
 from multiprocessing import Pool
 from dask.threaded import get as dask_threaded_get
-from dask.async import get_sync
+from dask.local import get_sync
```

Comment:
Yeah, it was preventing me from being able to run the `nldas_soil_moisture_ml.py` script. There's still a failure happening but I haven't gotten to the bottom of it yet:
```
Scoring - min 53.0741996765 11.031203472
Scoring - max 451.403808594 348.661744192
Scoring - mean 235.538560817 236.279549628
Scoring - MSE, R2, bounds 4794.19238344 0.392505454101 True
Scoring - std 69.6320856652 54.0152447214
Scoring - median 239.106201172 237.448162695
Scoring - min 53.0741996765 12.0512496036
Scoring - max 451.403808594 349.846799064
Scoring - mean 235.538560817 236.312058288
Scoring - MSE, R2, bounds 4790.09465289 0.393024696711 True
2017-07-17 14:09:01,811 - elm.pipeline.ensemble - ensemble:229 - Trained 40 estimators
Scores: [(2497.7730388236732, 0.6834954927511424, 1), (2497.774147841795, 0.6834953522222678, 1), (2497.774147841795, 0.6834953522222678, 1), (2497.77414784
2017-07-17 14:09:01,863 - elm.pipeline.predict_many - predict_many:117 - Predict 6 estimator(s) and 1 sample(s) (6 combination[s])
Traceback (most recent call last):
  File "nldas_soil_moisture_ml.py", line 556, in <module>
    last_hour_data, this_hour_data, models, preds, models2, preds2 = main()
  File "nldas_soil_moisture_ml.py", line 545, in main
    out = train_model_on_models(last_hour_data, this_hour_data, init_func)
  File "nldas_soil_moisture_ml.py", line 405, in train_model_on_models
    X_second = second_layer_input_matrix(X, models)
  File "nldas_soil_moisture_ml.py", line 343, in second_layer_input_matrix
    ensemble=models)
  File "/Users/gbrener/elm/elm/pipeline/predict_many.py", line 120, in predict_many
    new = dask.get(dsk, keys)
  File "/Users/gbrener/miniconda/envs/nldas_py27/lib/python2.7/site-packages/dask/local.py", line 562, in get_sync
    return get_async(apply_sync, 1, dsk, keys, **kwargs)
  File "/Users/gbrener/miniconda/envs/nldas_py27/lib/python2.7/site-packages/dask/local.py", line 529, in get_async
    fire_task()
  File "/Users/gbrener/miniconda/envs/nldas_py27/lib/python2.7/site-packages/dask/local.py", line 504, in fire_task
    callback=queue.put)
  File "/Users/gbrener/miniconda/envs/nldas_py27/lib/python2.7/site-packages/dask/local.py", line 551, in apply_sync
    res = func(*args, **kwds)
  File "/Users/gbrener/miniconda/envs/nldas_py27/lib/python2.7/site-packages/dask/local.py", line 295, in execute_task
    result = pack_exception(e, dumps)
  File "/Users/gbrener/miniconda/envs/nldas_py27/lib/python2.7/site-packages/dask/local.py", line 290, in execute_task
    result = _execute_task(task, data)
  File "/Users/gbrener/miniconda/envs/nldas_py27/lib/python2.7/site-packages/dask/local.py", line 271, in _execute_task
    return func(*args2)
  File "/Users/gbrener/elm/elm/pipeline/predict_many.py", line 38, in _predict_one_sample_one_arg
    prediction, X_final = estimator.predict(X, return_X=True)
  File "/Users/gbrener/elm/elm/pipeline/pipeline.py", line 490, in predict
    return self._run_steps(X, **kw)
  File "/Users/gbrener/elm/elm/pipeline/pipeline.py", line 185, in _run_steps
    pred = fitter_or_predict(X.flat.values, **kwargs)
TypeError: predict() got an unexpected keyword argument 'sample_weight'
```